en/docs/monitoring-and-insights/view-logs.md

```diff
@@ -0,0 +1,106 @@
+
+As of now, Choreo captures the following user-specific operations as audit logs:
+
+- Project creation, update, and deletion.
+- Component creation, update, and deletion.
+- Component promotion initiation.
+- Component version creation.
+- Component deployment, redeployment, and undeployment initiation for all components other than REST API Proxy components.
+- Component API access mode update.
+- Enabling and disabling component auto-deployment on commit. 
+- Component build configuration update.
+- Component endpoint creation, update, and deletion.
+- Organization user management.
+- On-premises key management.
+- Project-level configuration management.
+
+In Choreo, organization administrators are allowed to view audit logs by default. If other members need to access organization-specific audit logs, the administrator can create a role with the relevant permission and assign it to members. For step-by-step instructions on how to create and assign a role with relevant permission, see [Manage audit log access](#manage-audit-log-access).
+
+To view audit logs, follow these steps:
+
+1. Sign in to [Choreo](https://console.choreo.dev/) using your Google, GitHub, or Microsoft account.
+2. In the Choreo Console, go to the top navigation menu and click **Organization**.
+   
+    !!! tip
+         As of now, you can only view organization-level audit logs.
+
+3. In the left navigation menu, click **Logs** and then click **Audit Logs**. This displays audit logs for the past 30 days by default.
+
+    ![Audit logs](../assets/img/monitoring-and-insights/view-logs/audit-logs.png)
+
+    You can apply the necessary filter criteria to view logs according to your requirements.
+
+### Audit log retention
+
+Choreo retains audit logs for one year and archives them for an additional year. Therefore, the total retention period for audit logs is two years.
+
+### Manage audit log access
+
+Follow the steps given below to create a role with audit log access permission and assign it to organization members who need access to audit logs:
+
+!!! info "Note"
+        You must be the organization administrator to perform this action.
+
+#### Step 1: Create a role with audit log access permission
+
+1. In the Choreo Console, go to the top navigation menu and click **Organization**.
+2. In the left navigation menu, click **Settings**.
+3. On the **Organization** tab, click **Roles** and then click **+ Create Role**.
+4. Enter a name and description for the role.
+   
+     ![Create role](../assets/img/monitoring-and-insights/view-logs/create-role-to-view-audit-logs.png)
+
+5. Click **Next**.
+6. In the **Create Role** dialog, select **LOG-MANAGEMENT** under **Permissions**.
+
+     ![Select log management permission](../assets/img/monitoring-and-insights/view-logs/log-management-permission.png)
+
+7. Click **Create**.  
+
+
+#### Step 2: Assign the created role to an organization member who needs to have access to audit logs
```

Comment:
Shall we just say, Assign the created role to an organization member?

Reply:
+1
Done.